tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/reindexing/ElasticsearchSiteIndexingTests.java

```diff
@@ -103,14 +112,23 @@ public void testSiteUseCasesForCreateModifyDeleteSite()
         // Remove the automatically created Sample Site
         deleteSite(SAMPLE_SITE_ID);
 
+        //Sometimes this test may fail, so if previous data exists it must be deleted before the next run
```

Comment:
How does this test's retry happen exactly?

As far as I can see the Elasticsearch TestNG suite that is triggering the `reindexing` tests does not use the `org.alfresco.elasticsearch.parallel.RetryAnnotationTransformer` that kicks in the `RetryAnalyzer`: https://github.com/Alfresco/acs-packaging/blob/e00c708955af7733decb72b2ccf3254bc5a4167f/tests/tas-elasticsearch/src/test/resources/test-suites/elasticsearch-suite.xml#L5.

Reply:
I think that these tests use the `elastic` profile set up in `pom.xml` file:
https://github.com/Alfresco/acs-packaging/blob/b620a647bd03a44056675463bdf2a0b089beb105/tests/tas-elasticsearch/pom.xml#L19
And inside `standard-elasticsearch-suites.xml` there is `elasticsearch-parallel-suite.xml` which has `<listener class-name="org.alfresco.elasticsearch.parallel.RetryAnnotationTransformer" />`

It looks from the logs like this:
![error](https://github.com/Alfresco/acs-packaging/assets/115808064/431fddef-d2ad-4c01-be49-56a4e4a65f30)

Reply:
Seems to be a bug/unwanted behaviour of TestNG when merging test suites in one combined XML file: https://sqa.stackexchange.com/questions/37457/testng-listener-from-one-xml-file-is-applied-to-another.

So it was most likely unwanted/unknown by the person that introduced this listener + merged suite file. Either way, if it helps with making the tests a bit more stable I'm not against keeping it.

Reply:
Yes it helps a lot in this case

Reply:
cc @tpage-alfresco so he's aware about this behaviour _(in case he wasn't, I definitely wasn't)_ 👍